OSPSuite.Presentation.Importer/Services/IImporter.cs

```diff
@@ -0,0 +1,51 @@
+﻿using OSPSuite.Presentation.Importer.Core;
+using System.Collections.Generic;
+using System.Linq;
+using IoC = OSPSuite.Utility.Container.IContainer;
+
+namespace OSPSuite.Presentation.Importer.Services
+{
+   public interface IImporter
+   {
+      IDataSourceFile LoadFile(string path);
+      IDataSource ImportFromFile(IEnumerable<IDataSheet> sheets);
+      IList<IDataFormat> AvailableFormats(IUnformattedData data);
+   }
+
+   public class Importer : IImporter
+   {
+      private readonly IoC container;
+      private readonly IDataSourceFileParser parser;
+
+      public Importer(IoC container)
+      {
+         this.container = container;
+         this.parser = container.Resolve<IDataSourceFileParser>();
```

Comment:
Why not have the parser as dependencies?

Reply:
We already have the container as a dependency which enable us to resolve it... but yes, it will be more flexible adding it as a dependency